### DISCUSSION TITLE
无法更改显示字体为pc安装过的字体 Unable to change display font to one installed on pc

### DISCUSSION THREAD

**Discussion by LCH233-233** [ClassIsland/ClassIsland#214] [Q&A] [answered]
软件默认字体是Harmony OS Sans，我自己下载安装了HONOR Sans CN并安装到电脑上字体库里，**但是当我打开这个软件时无法在选择自己里面选择此字体。**
![屏幕截图 2024-07-15 174625](https://github.com/user-attachments/assets/aec1761d-7981-492d-b11e-3ea60ec63927)
![屏幕截图 2024-07-15 174716](https://github.com/user-attachments/assets/dd46f1a3-6d92-4876-82a8-1d062b0ee0a1)
![sndjdnsdsm 174758](https://github.com/user-attachments/assets/ac6cc4a5-a0ac-4e3f-ab59-6ef23bedbc49)
如图，G后面是I，而不是理应的“H”
没有试过测试版软件，但是我看过了别的issue并没有发现别人发布相关问题

**Comment by LCH233-233** [comment]
![屏幕截图 2024-07-15 175238](https://github.com/user-attachments/assets/3cd80b37-99cb-4080-958b-362e7fe38d48)
![屏幕截图 202dsss 175030](https://github.com/user-attachments/assets/dd3b3688-dd3c-4b90-9466-c9972a5d479e)
![屏幕截图 2024-07-15 174937](https://github.com/user-attachments/assets/af39d16f-7a3a-4de3-9976-0d9e259b8870)

**Comment by DryIce-cc** [comment, marked as answer]
重启 ClassIsland 试试。
字体列表可能不会按照首字母排序，有可能会在列表最上面或者别的地方。

**Reply by LCH233-233** [reply]
非常感谢，问题已经解决了
